### PR TITLE
Created editor component

### PR DESCRIPTION
**Pull request by sodabarauf**
this is the third react project here i did these works:
-added react app
-follow GitHub workflow
-added tags and functions to app.js root component
-added styles
-deleted unnecessary files
-added descriptive readme.md file
-added linters

### CODE REVIEW

**Comment by humuyar** [comment]
hi @sodabarauf jab i review it u done it perfect but change the readme file content bcuz it show me the app read me

File: README.old.md
Comment (ahzamir) [inline README.old.md:1]: - Move this file contents to the README.md file and then remove this you only need one README.md file on your project

**Review by ahzamir — APPROVED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Markdown Previewer project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.old.md
Comment (sodabarauf) [reply README.old.md:1]: @ahzamir 
I appreciate from all your supports and feedback thank you sir.
ok i will do this.

**Pull request merged**